api/app/utils/redapp.py

```diff
@@ -407,6 +438,7 @@ def FBPCalculateStatisticsCOM(elevation: float,  # pylint: disable=invalid-name,
         fbp.crownBase = crown_base_height
 
         fbp.useSlope = False
+        fbp.useCrownBaseHeight = use_crown_base_height
 
         # TODO: Sybrand figure out timezone
```

Comment:
does Sybrand still need to figure out timezone as part of this ticket, or is that a separate PR?

Reply:
I'll do it as part of this one.